diff --git a/node_modules/hardhat/internal/hardhat-network/provider/node.js b/node_modules/hardhat/internal/hardhat-network/provider/node.js
index 17c5dd2..fc28da9 100644
--- a/node_modules/hardhat/internal/hardhat-network/provider/node.js
+++ b/node_modules/hardhat/internal/hardhat-network/provider/node.js
@@ -1590,7 +1590,9 @@ Hardhat Network's forking functionality only works with blocks from at least spu
     }
     _newDeadline() {
         const dt = new Date();
-        dt.setMinutes(dt.getMinutes() + 5); // This will not overflow
+
+        // Patch: Extend default deadline (5 minutes) such that filters won't be deleted as quickly
+        dt.setDate(dt.getDate() + 1); // This will not overflow
         return dt;
     }
     _getNextFilterId() {
